bundle/rhpam/metadata/annotations.yaml

```diff
@@ -0,0 +1,15 @@
+annotations:
+  # Core bundle annotations.
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  operators.operatorframework.io.bundle.metadata.v1: metadata/
+  operators.operatorframework.io.bundle.package.v1: rhpam-kogito-operator
+  operators.operatorframework.io.bundle.channels.v1: alpha,1.x
+  operators.operatorframework.io.bundle.channel.default.v1: 1.x
```

Comment:
Channels here are using community values, in current RHPAM operator there is `7.x`.

Reply:
done